Update advice provided to Falcon signature verification

Al-Kindi-0 opened the pull request:
This is parallel to https://github.com/0xPolygonMiden/miden-vm/pull/1661

bobbinth commented:
@Al-Kindi-0 - to make the tests pass here, we need to update this PR to use the latest version of Miden VM (i.e., the `next` branch) - right? Or is there anything else outstanding?

Al-Kindi-0 commented:
I don't think so, but this was dependent on the VM PR being merged which is now the case.
Went ahead and updated the dependencies but it seems like the bus is failing. I thought we fixed it and I am not sure what would cause this from the recent changes related to the current PR.

bobbinth commented:
> Went ahead and updated the dependencies but it seems like the bus is failing. I thought we fixed it and I am not sure what would cause this from the recent changes related to the current PR.

Ah interesting! @plafer - could you take a look at the bus issue?

plafer commented:
Opened a PR with a fix in the VM repo: https://github.com/0xPolygonMiden/miden-vm/pull/1689

Al-Kindi-0 commented:
All good now!

bobbinth commented:
@Al-Kindi-0 - I think this can now be updated to point to `miden-vm`'s `next` branch, right?

One thing I'm not sure of is whether this needs to be first merged into #1214 (or vise-verse) first.

Al-Kindi-0 commented:
> @Al-Kindi-0 - I think this can now be updated to point to `miden-vm`'s `next` branch, right?
> 
> One thing I'm not sure of is whether this needs to be first merged into #1214 (or vise-verse) first.

This PR is relatively smaller compared to #1214 so it makes sense to merge it in #1214.

Al-Kindi-0 commented:
I have now rebased on `santiagopittella-rust-2024-edition` and changed the PR base
